### ISSUE TITLE
Wrongly fails on modifier

### ISSUE THREAD

**Issue by NSchweers** [stumpwm/stumpwm#213]
Using the layout described in http://neo-layout.org and http://wiki.neo-layout.org/wiki/Bone, stumpwm fails on C-t ?, because it is necessary to push ISO_Level3_Shift, in order to reach ?.  It it possible to make stumpwm ignore a key, like ISO_Level3_Shift?

**Comment by Harleqin** [comment]
I have the same problem (I am using Neo).  I had briefly tried to look into it, but I do not yet understand enough of how X works.  Any pointers might help.

**Comment by dbjergaard** [comment]
Stumpwm doesn't support nonstandard keyboard layouts.  When I say doesn't I mean, technically not at all even though there is some plumbing in the code that allows partial support.  I'm not sure what it would take to work around your issue.

**Comment by Harleqin** [comment]
I would have imagined that X translates keyboard input into characters and control modifiers, but looking at what stumpwm seems to get, it's not that easy.

**Comment by dbjergaard** [comment]
I'm not sure where you're looking, but the places you want to start are keytrans.lisp and kmap.lisp

**Comment by dbjergaard** [comment]
can you try the fix proposed in #194? specifically:
```
xkbcomp $DISPLAY - | egrep -v "group . = AltGr;" | xkbcomp - $DISPLAY
```

**Comment by dbjergaard** [comment]
If this issue is still affecting people and the proposed fix in #194 doesn't work, please reopen.

**Comment by vherrmann** [comment]
The proposed fix didn't work for me :/
